index.html

```diff
@@ -4,44 +4,69 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  
     <title>Sabka Bazar</title>
-    <!-- <base href="/"> -->
+
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
-    <link type="text/css" rel="stylesheet" href="./shared/css/style.css">
     <script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
-   
-    <script src="src/app/signup/signup.js"></script>
-    <script src="src/app/login/login.js"></script>
-    <script src="src/app/home/home.js"></script>
-    <script src="src/app/products/products.js"></script>
-    <script src="src/app/index.js"></script>
   </head>
   <body>
-    <header class="header">
+    <header>
       <nav class="navigation">
-        <img src="./assets/images/logo.png" alt="Sabka Bazar logo" class="logo" onclick="onNavItemClick('home')">
-        <ul class="navbar">
-          <li><a href="#" onclick="onNavItemClick('home')">Home</a></li>
-          <li><a href="#" onclick="onNavItemClick('products')">Products</a></li>
-        </ul>
+        <img src="images/logo.png" alt="Sabka Bazar logo" class="logo">
+        <div class="navbar">
+          <a href="#" class="nav-home">Home</a>
+          <a href="#" class="nav-product">Products</a>
+        </div>
         <div class="action-buttons">
-          <a href="#" class="btn-signin" onclick="onNavItemClick('signin')">SignIn</a>
-          <a href="#" class="btn-register" onclick="onNavItemClick('signup')">Register</a>
+          <a href="#" class="btn-signin">SignIn</a>
+          <a href="#" class="btn-register">Register</a>
         </div>
         <div class="cart">
-          <img class="cart-icon" src="assets/images/cart.svg">
-          <span class="items-in-cart">0 items</span>
+          <a href="#">
+            <img class="cart-icon" src="images/cart.svg">
+            <span class="items-in-cart">0</span>
+            <span class="items-text">&nbsp;items</span>
+          </a>
+          <span class="right-nav" onclick="onDrawerIconClick()">&#9776;</span>
+        </div>
+      </nav>
+      <nav>
+        <div class="drawer-nav" id="drawer">
+          <a href="#" class="cross-icon" onclick="onDrawerCloseIconClick()">&times;</a>
+          <a href="#" class="nav-home">Home</a>
+          <a href="#" class="nav-product">Products</a>
```

Comment:
Why some of the NAV element are coming twice? such duplication should be avoided.